tests/decode/test3_simple/CMakeLists.txt

```diff
@@ -40,8 +47,12 @@ execute_process(
   )
 
 include(${PROJECT_BINARY_DIR}/pet.cmake)
-include(${PROJECT_BINARY_DIR}/serial_recovery.cmake)
+
+zephyr_include_directories(${PROJECT_BINARY_DIR}/include
+  ${CMAKE_CURRENT_LIST_DIR}/../../../include)
+zephyr_sources(${PROJECT_BINARY_DIR}/src/serial/serial_recovery_decode.c
+  ${CMAKE_CURRENT_LIST_DIR}/../../../src/zcbor_decode.c
+  ${CMAKE_CURRENT_LIST_DIR}/../../../src/zcbor_common.c)
```

Comment:
As this is zephyr specific, should we use `${ZEPHYR_ZCBOR_MODULE_DIR}/src/` instead of `../`-ing out

Reply:
The tests don't always run with zcbor as a module, so this var is not always available.